returnn/datasets/sprint.py

```diff
@@ -340,6 +343,8 @@ def add_new_data(self, features, targets=None, segment_name=None):
     # is in format (feature,time)
     assert self.num_inputs == features.shape[0]
     num_frames = features.shape[1]
+    # features maybe downsampled via the network
+    reduce_num_frames = int((num_frames + self.reduce_target_factor - 1) / self.reduce_target_factor)
```

Comment:
would it make sense to make the rounding configurable?

Reply:
This is a good default though, as you almost always have ceildiv, eg in pool, conv, etc. I'm not sure that you ever have sth else. So maybe we don't need to add it until we really need it.

Reply:
I had the case where the model (conformer) would do internal downsampling of factor 3, then upsample to FFR, then I downsample factor 2 to generate alignments.

Now depending on T%6 some weird edge cases arised. But I agree that this use case is contrived and that cieldiv is usually a good solution.

We can keep it like this, I only wanted to spring a discussion.

@DanEnergetics this is maybe also interesting for you.

Reply:
The conv layer seems to give different output dimensions depending on the [selected padding](https://github.com/rwth-i6/returnn/blob/ce4366ff0caafc2e4b349fd2a189870f3c76f630/returnn/tf/layers/basic.py#L5228). This would cover the case that padding "SAME" is used but not "VALID" in all cases if I'm not mistaken. The pool layer probably behaves similar. Is "SAME" then more common or better in general?
Apart from that, this seems like a really useful change.